getting_started/scripting/c_sharp/c_sharp_style_guide.rst

```diff
@@ -298,9 +295,10 @@ Other considerations
 
  * Use explicit access modifiers.
  * Use properties instead of non-private fields.
- * Use modifiers in this order: *'public/protected/private/internal virtual/override/abstract/new static readonly'*.
- * Avoid using fully qualified names or *'this.'* prefix for members when it's not necessary.
- * Remove unused *'using'* statements and unnecessary parentheses.
- * Consider omitting default initial value for a type.
+ * Use modifiers in this order:
+   ``public``/``protected``/``private``/``internal virtual``/``override``/``abstract``/``new static readonly``.
```

Comment:
The last `new static readonly` should be 3 separate modifiers.

Reply:
Done. Should `internal virtual` be separated as well?

Reply:
Yes, didn't notice that one.